core/src/main/java/org/frankframework/management/bus/endpoints/ServerStatistics.java

```diff
@@ -227,7 +227,7 @@ private List<Object> mapMessageKeeperMessages(MessageKeeper messageKeeper) {
 			}
 			configurationMessage.put("message", msg);
 			Instant date = messageKeeper.getMessage(t).getMessageDate();
-			configurationMessage.put("date", DateFormatUtils.format(date, DateFormatUtils.FULL_GENERIC_FORMATTER));
+			configurationMessage.put("date", Date.from(date));
```

Comment:
Dit was een (geformatte) string maar is nu een date, hoe ziet het formaat er nu uit?

Reply:
Ja het is steeds ook het zelfde object, beetje stom dat we het dan anders mappen.. een DTO zou hier al bij helpen. Maar hopelijk werken we daar nu langzaam naar toe... #9298 & #9304 (more to come).